_activities/pyhep.md

```diff
@@ -321,6 +319,8 @@ The event was kindly sponsored by
 
 
 # Previous conveners
+- Nikolai Krug (ATLAS and Belle II, LMU Munich), 2023-2025
+- Matthew Feickert (ATLAS, University of Wisconsin-Madison), 2023-2025
```

Comment:
I'm not 100% sure on the convener dates for us. In my CV I have January 2023, but I don't remember when I moved from organizing to being a convener. I'm assuming this is correct.